FinalTask/FinalTaskGM/FinalTaskGM/PageObject/BasePage.cs

```diff
@@ -0,0 +1,26 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+
+namespace FinalTaskGM.PageObject
+{
+    public class BasePage
+    {
+        private readonly IClock clock = new SystemClock();
+
+        public void WaitUntilElementIsDisplayed(By locator, IWebDriver driver)
```

Comment:
This method should has bool return type. And then you will be able to use this method together with asserts.